src/sentry/tasks/integrations.py

```diff
@@ -18,3 +19,14 @@ def post_comment(external_issue_id, data, **kwargs):
     external_issue = ExternalIssue.objects.get(id=external_issue_id)
     integration = Integration.objects.get(id=external_issue.integration_id)
     integration.get_installation().create_comment(external_issue.key, data['text'])
+
+
+@instrumented_task(
+    name='sentry.integrations.jira.sync_metadata',
```

Comment:
Correct, the name doesn't _really_ many anything, except for identifying the task. But per convention, we try to keep it to the import path. But this isn't strictly true since they may move around, in which case, we don't rename the task.